cmd/cdi-importer/importer.go

```diff
@@ -49,8 +48,20 @@ func main() {
 	source, _ := util.ParseEnvVar(common.ImporterSource, false)
 	contentType, _ := util.ParseEnvVar(common.ImporterContentType, false)
 	imageSize, _ := util.ParseEnvVar(common.ImporterImageSize, false)
+	volumeMode, _ := util.ParseEnvVar(common.VolumeMode, false)
 
 	dest := common.ImporterWritePath
+	if volumeMode == "Block" {
```

Comment:
We should also be doing this check in the validating web hook.

Reply:
Do we want to do it even if it's temporary solution? We are going to have FS PVC in phase II of this feature, so I wonder if we can skip the webhook validation..... or do it and delete it when phase II is done..